tests/Makefile.am

```diff
@@ -82,6 +82,7 @@ check-headers: $(auto_c_files:.c=.o)
 
 distclean-compile:
 	rm -rf auto_*.c
+	rm -rf .deps
```

Comment:
Looks to me the target should rather be named `distclean-local`,
cannot find any support for this one to be supported, hence we
should avoid surprises with the gray zone.

Reply:
interesting! All these built-in destinations still throw me, and some are hard to find in the doc. disclean-local works for me so I''m happy to change it.

Is there a good reference for these things? the info files seem to be unhelpful.

Reply:
https://www.gnu.org/software/automake/manual/automake.html#Clean
(as applicable to many other cases incl. pacemaker: single-page compilation,
finding through the page, even though info pages map 1:1 for most GNU
projects, IIRC)